### PR TITLE
fix: lock orientation to "portrait"

### PR DESCRIPTION
**Pull request by dimatretyak**
### Description
See more context here: https://artsy.slack.com/archives/C02BAQ5K7/p1663316451086229

#### Before
https://user-images.githubusercontent.com/3513494/190615758-990a29a7-8d00-4c5f-aaeb-04518d1b8ed5.mp4

#### After
https://user-images.githubusercontent.com/3513494/190615796-858d445c-76b5-4c13-94e8-eeb943d45dee.mp4

### Tablet
https://user-images.githubusercontent.com/3513494/190617238-5f8fa707-e7dd-4446-8be8-f6b269a5c86b.mp4

This PR resolves []

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- lock orientation to "portrait" - dimatretyak

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/navigation/NavStack.tsx [hunk 1]
```diff
@@ -86,6 +86,7 @@ export const NavStack: React.FC<{
       screenOptions={{
         headerShown: false,
         contentStyle: { backgroundColor: "white" },
+        orientation: "portrait",
```
Comment (dimatretyak) [inline src/app/navigation/NavStack.tsx:89]: **question**: should we allow "portrait" and "landscape" orientation for tablets, as we did before?

chore: the chorest chore of all: reanimated upgrade (third try, metav… · artsy:eigen@7bfe84c 2022-09-16 12-45-16

**Review by gkartalis — APPROVED** [review]

File: src/app/navigation/NavStack.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/navigation/NavStack.tsx:89]: I think we should

**Review by pvinis — APPROVED** [review]

File: src/app/navigation/NavStack.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/app/navigation/NavStack.tsx]: got it. done!

**Review by MounirDhahri — APPROVED** [review]
Nice - thanks Dima for fixing this

**Pull request merged**